main.go

```diff
@@ -18,10 +18,9 @@ package main
 
 import (
 	"flag"
+	"github.com/AlonaKaplan/kubesecondarydns/pkg/controllers"
```

Comment:
our include should be in the last section of the import i think

Reply:
done

Reply:
doesnt seem so ;]

Reply:
done